db/migrate/20200930155458_create_answers.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAnswers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :answers do |t|
+      t.integer :question_id
```

Comment:
原則として，外部キー設定にしておいた方がよいですね。

```
rails db:rollback
```

を実行した上で，次のように修正しましょう。

```suggestion
      t.references :question, null: false, foreign_key: true
```

そして再度マイグレーションを行ってプッシュして下さい。

```
rails db:migrate
```